tests/features/content_types/branch.feature

```diff
@@ -16,3 +42,18 @@ Feature: Branch Content type
       | Phone          | +1234567890    |
     When I press "Save and publish"
     Then I should see the message "Branch Branch One has been created."
+
+  Scenario: I see appropriate content on branch
+    Given I view node "behat_branch"
+    Then I should see "BEHAT BRANCH"
+    And I should see "Main road 10"
+    And I should see "98101"
+    And I should see "WA"
+    And I should see "Seattle"
+    And I should see "+1234567890"
+    And I should see "BEHAT BANNER 01"
+    And I should see "BEHAT DESCRIPTION 01"
+    And I should see "BEHAT BANNER 02"
+    And I should see "BEHAT DESCRIPTION 02"
```

Comment:
Is it possible to define some areas. So that we can verify where it actually appear.
I think locators can be based on grid columns and other css properties.